CONFIGURATION.md

```diff
@@ -53,6 +53,11 @@ These environment variables are required for all services.
 | RELAYER_SCREENER_TOKEN | Authorization token for screener service | string |
 | RELAYER_BLOCKED_COUNTRIES | A list of country codes delimited by spaces | string|
 | RELAYER_EXPRESS_TRUST_PROXY | If set to `true`, then `trust proxy` express setting will be enabled. Defaults to `false` | boolean |
+| RELAYER_FEE_MANAGER_TYPE | Fee manager that will be used for fee estimations | FeeManagerType |
+| RELAYER_FEE_SCALING_FACTOR | Scaling factor (in percent) used to adjust fee validation. Defaults to 100. | number |
```

Comment:
This way we cannot "scale" user fees, since we always expect them to pay the amount approximately equal to the cost of gas. Let's rename this option to `RELAYER_FEE_MARGIN_FACTOR`, and use second `RELAYER_FEE_SCALING_FACTOR` parameter to proportionally scale results of both: `estimateFee` and `getFees` functions.

So that configuration like:
```
RELAYER_FEE_MARGIN_FACTOR=90
RELAYER_FEE_SCALING_FACTOR=50
```
would mean that the relayer wants to subsidize ~50% of gas costs with 10% margin.

While the configuration like:
```
RELAYER_FEE_MARGIN_FACTOR=90
RELAYER_FEE_SCALING_FACTOR=150
```
would mean that the relayer wants to charge extra 50% on top of its gas costs.

Reply:
Done